### PR TITLE
Guide for running Gardener on Minikube

### PR DESCRIPTION
**Pull request by mvladev**
The following scenario is supported - running the Gardener control plane components in Minikube and running the vagrant-provider on the host machine for creation of Shoot clusters locally. This setup simplifies the initial deployment setup.

@vasu1124 @rolpat can you test this setup on your machines?

### CODE REVIEW

**Review by rfranzke — APPROVED** [review]

**Comment by rfranzke** [comment]
@mvladev have approved, will wait for approval of @vasu1124 or @rolpat before final merge.

**Comment by rfranzke** [comment]
ping @vasu1124 @rolpat, could you please give feedback so that we can go ahead with this PR?

File: docs/deployment/kubernetes-minikube.md
Comment (vasu1124) [inline docs/deployment/kubernetes-minikube.md]: typo: Deploying

**Comment by rfranzke** [comment]
Thanks @vasu1124. @mvladev, squash please.

File: charts/gardener/vagrant-local-values.yaml
Comment (vasu1124) [inline charts/gardener/vagrant-local-values.yaml]: 0.2.0-f15944037e1784bada87bd4bff51b5a334b3b066

File: charts/gardener/vagrant-local-values.yaml
Comment (vasu1124) [inline charts/gardener/vagrant-local-values.yaml]: 0.2.0-f15944037e1784bada87bd4bff51b5a334b3b066

**Review by vasu1124 — COMMENTED** [review]
looks good to me. works on my machine w/ new tags. thanks to rafael.

**Comment by mvladev** [comment]
Squashed and changed gardener version to `0.2.0-f15944037e1784bada87bd4bff51b5a334b3b066`

**Review by rfranzke — APPROVED** [review]

**Comment by rfranzke** [comment]
@mvladev want to merge, but you need to rebase on `master`, please.

**Comment by mvladev** [comment]
Rebased

**Pull request merged**
